crypto/Cargo.toml

```diff
@@ -24,8 +24,11 @@ serde                       = { workspace = true, features = ["derive"] }
 thiserror.workspace         = true
 
 [dev-dependencies]
-rand.workspace = true
-rand_chacha    = "0.3"
+quickcheck.workspace        = true
+quickcheck_macros.workspace = true
+rand.workspace              = true
+rand_chacha                 = "0.3"
```

Comment:
I don't think you need `rand` and `rand_chacha`. Must have been a leftover from earlier times.